rust/kernel/print.rs

```diff
@@ -132,6 +133,7 @@ pub unsafe fn call_printk(
     args: fmt::Arguments<'_>,
 ) {
     // `_printk` does not seem to fail in any path.
+    // SAFETY: FFI call.
```

Comment:
I don't think this justification is sufficient. What prevents one from passing an unsafe combination of arguments to `_printk`?

(I understand you may not want to figure it out, or write it down here, which would be fine. But in that case, I'd rather leave this unsafe block without a SAFETY comment than with a clearly incomplete one.)

Reply:
(As a hint, check the safety contract of this function)